src/FunctionInfo.cs

```diff
@@ -24,9 +24,12 @@ internal class AzFunctionInfo
         private const string ActivityTrigger = "activityTrigger";
 
         internal const string TriggerMetadata = "TriggerMetadata";
+        internal const string TraceContext = "TraceContext";
```

Comment:
Please also update the design doc (https://github.com/Azure/azure-functions-powershell-worker/blob/dev/docs/designs/PowerShell-AzF-Overall-Design.md) as well: just a short note describing this parameter, perhaps with a usage sample.

Reply:
Updated InvocationRequest section with some text and snippet of how to access TraceContext. Please let me know if anything more needs to be added there.